clarity_ext/repository/step_repository.py

```diff
@@ -171,5 +174,9 @@ def current_user(self):
         "target_concentration": "Target Concentration",
         "target_volume": "Target Volume",
         "volume": "Current sample volume (ul)"
+    },
+    "ResultFile": {
```

Comment:
Here, I again think this is not really something that has to do with ResultFiles in general, but this ResultFile you're currently working with, while the other map had to do with Analytes.

I think you should probably think about having an object that inherits from ResultFile which has these properties.